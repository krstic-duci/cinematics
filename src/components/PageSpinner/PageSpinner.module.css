.pageSpinnerContainer {
  align-items: center;
  background-color: rgba(13, 37, 63, 0.1);
  display: flex;
  height: 100%;
  justify-content: center;
  position: fixed;
  width: 100%;
  z-index: 10;
}
