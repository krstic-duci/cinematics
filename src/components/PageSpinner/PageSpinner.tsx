import Spinner from "react-bootstrap/Spinner";

import styles from "./PageSpinner.module.css";

const PageSpinner = () => (
  <div className={styles.pageSpinnerContainer}>
    <Spinner animation="border" />
  </div>
);

export default PageSpinner;
