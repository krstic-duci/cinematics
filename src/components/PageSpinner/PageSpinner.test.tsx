import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";

import PageSpinner from "./PageSpinner";

describe("PageSpinner component", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(
      <MemoryRouter>
        <PageSpinner />
      </MemoryRouter>
    );
    expect(tree).toMatchSnapshot();
  });
});
