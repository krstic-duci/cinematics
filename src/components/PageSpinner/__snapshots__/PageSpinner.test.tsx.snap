// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSpinner component should match snapshot 1`] = `
<div
  className="pageSpinnerContainer"
>
  <div
    className="spinner-border"
  />
</div>
`;
